pgmq-extension/sql/pgmq.sql

```diff
@@ -282,6 +282,29 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+-- send_at
+-- sends a message to a queue, with a delay as a timestamp
+CREATE FUNCTION pgmq.send_at(
+    queue_name TEXT,
+    msg JSONB,
+    delay TIMESTAMP
+) RETURNS SETOF BIGINT AS $$
```

Comment:
Will it work if we overload the functions instead of defining new function names?

```suggestion
CREATE FUNCTION pgmq.send(
    queue_name TEXT,
    msg JSONB,
    vt TIMESTAMPZ
) RETURNS SETOF BIGINT AS $$
```

Reply:
I figured it would make more sense to be consistent between sql and python since python doesn't have function overloading

Reply:
The extension is not aware of which clients exist, so I don't want to make assumptions in the extensions that depend on clients.

However, we'd not really overloading much elsewhere so let's go ahead and keep this `send_batch()`. I will likely put together a proposal to overload everything for the pgmq v2 release though.